src/cow-react/modules/swap/hooks/useFlowContext.ts

```diff
@@ -65,7 +66,8 @@ interface BaseFlowContextSetup {
   ensRecipientAddress: string | null
   allowsOffchainSigning: boolean
   swapConfirmManager: SwapConfirmManager
-  isEthFlow: any
+  isEthFlow: boolean
+  isSafeBundle: boolean
```

Comment:
the reason is that they are 2 booleans is because they both can-coexist? could they be both true? If they are mutually exclusive we might need to use an enum instead